app/models/project.rb

```diff
@@ -2,6 +2,7 @@
 
 class Project < ApplicationRecord
```

Comment:
we'll also need to ensure a SchoolProject is created every time a Project is, we could either do this in a hook like `after_create`, though the logic to ensure it only happens for schools might be trickier to implement here, or do it in Projects concept, and maybe add a validation (though that might negate avoiding using a hook since the logic will be similarish to validate)